sync/config/pipelines.yaml

```diff
@@ -26,8 +26,8 @@ archive: https://github.com/tektoncd/pipeline/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
-- name: release-v0.54.x
-  displayName: v0.54.x
+- name: release-v0.56.x
```

Comment:
This branch doesn't exist in tektoncd/pipeline 👀

Reply:
Ah Thanks for the catch. I had named it `...v0.56.0` I just renamed it to `...v0.56.x`

Reply:
Is that why the tests below were failing?

Reply:
Yes that's the reason